pkg/universe.dagger.io/x/berryphillips@gmail.com/onepassword/example/plan.cue

```diff
@@ -0,0 +1,57 @@
+package example
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/x/berryphillips@gmail.com/onepassword"
+)
+
+dagger.#Plan & {
+	client: env: {
+		OP_ADDRESS:    dagger.#Secret
+		OP_EMAIL:      dagger.#Secret
+		OP_SECRET_KEY: dagger.#Secret
+		OP_PASSWORD:   dagger.#Secret
+	}
+
+	_opCreds: onepassword.#Credentials & {
+		address:   client.env.OP_ADDRESS
+		email:     client.env.OP_EMAIL
+		secretKey: client.env.OP_SECRET_KEY
+		password:  client.env.OP_PASSWORD
+	}
+
+	actions: {
+		image: core.#Pull & {source: "alpine:3"}
```

Comment:
It would look something like this:

```
import (
  "universe.dagger.io/alpine"
)

// Build an alpine base image
image: alpine.#Build

example: core.#Exec & {
  input: image.output.rootfs
}
```

Note: `alpine.#Build` produces a `docker.#Image`. If you want to use that directly, instead of accessing the raw rootfs, consider using `docker.#Run`. Both are fine to use, the difference is that one has more high-level conveniences built-in than the other.

Reply:
`.rootfs` 🤦🏻‍♂️ But I do prefer the suggestion of `docker.#Run` as that better describes the action. Thanks for the example and suggestions!